info.json

```diff
@@ -1,17 +1,19 @@
 {
-  "name": "colorblind_ultimate",
-  "version": "2.1.0",
-  "title": "Color Blind Ultimate",
-  "author": "rycieos",
-  "factorio_version": "2.0",
-  "dependencies": [
-      "? base >= 2.0.7",
-      "? elevated-rails >= 2.0.7",
-      "? quality >= 2.0.7",
-      "? space-age >= 2.0.7",
-      "? Ultracube >= 0.2.0"
-  ],
-  "description": "Ultimate options for color blindness accessibility. Includes custom icons, sprites, icon and sprite overlays, text overlays, customizable wire colors, customizable game and map overlay colors, and customizable technology background colors. Everything is individually customizable. Can be removed without affecting save games.",
-  "homepage": "https://github.com/Rycieos/factorio-colorblind-ultimate",
-  "contact": "https://github.com/Rycieos/factorio-colorblind-ultimate/issues/new/choose"
+    "name": "colorblind_ultimate",
+    "title": "♿Color Blind Ultimate ♿",
+    "description": "♿ Ultimate options for color blindness accessibility. \n  Includes custom icons, sprites, icon and sprite overlays, text overlays, customizable wire colors, customizable game and map overlay colors, and customizable technology background colors. Everything is individually customizable. \n Can be removed without affecting save games.",
+
+    "author": "rycieos",
+    "homepage": "https://github.com/Rycieos/factorio-colorblind-ultimate",
+    "contact": "https://github.com/Rycieos/factorio-colorblind-ultimate/issues/new/choose"
+
+    "version": "2.1.1",
+    "factorio_version": "2.0",
+    "dependencies": [
+    "base",
```

Comment:
Why remove the optional flag from this dependency? I realize that usually it is not possible to disable the base mod, but it is technically possible, and my mod is designed to be able to function without it.

Reply:
il n'est pas possible de lancer factorio sans factorio, donc lancer le mod sans la base de factorio est un non sense, de plus la standardisation de [base] implique de l'ecrire comme ceci.
techniquement cela devrai etre ? [base] mais je n'ai aucune idée de pourquoi les dev de factorio nous demande de l'ecrir ainsi.

Reply:
Incorrect. See [this Reddit post from Klonan](https://github.com/Rycieos/factorio-colorblind-ultimate/discussions/6#discussioncomment-11199482):

> Base mod is a 'mod' in the context of Factorio, in that it adds or modifies prototype data.
>
> If you were to add all the required prototypes (There are some that the game can't run without) in another mod, then you could play without base mod enabled at all.

While I am not aware of any mod that currently does this, it is possible. And like I said, my mod is designed to be able to function without base activated, so there is no reason to force it to be.